main.tf

```diff
@@ -98,6 +98,8 @@ resource "azurerm_kubernetes_cluster" "main" {
     }
   }
 
+  open_service_mesh_enabled = var.enable_open_service_mesh
+    
```

Comment:
@nlamirault you introduced a line with empty spaces at line 102. We are going to introduce very soon a CI that will lint the code, and this file will not pass the linting. Could you please remove the empty spaces ?

To avoid this pain in the future you can set your editor to automatically trim empty spaces at the end of lines when you save the files. Thank you

Reply:
Sorry :-1:  I made the fix using Github UI ... I fix that asap.

Reply:
Hi @zioproto agreed but I think we can accept this pr for this time since we'll submit a single pr to fix all the naming issues and style issues.